content/home/home-content.md

```diff
@@ -17,6 +20,14 @@ EPA is overseen by two congressional committees, the House Energy and Commerce C
 
 We want clean air, clean water, and an end to environmental racism and injustice. 
 
-**Environmental Enforcement Watch** is series of networked online workshops that bring people together from different backgrounds to analyze the data from Environmental Protection Agency’s <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws, facility inspections, and enforcement actions (or lack thereof) issued by the EPA and state agencies.
+We believe in the public’s right to know, and that we can collaboratively design public open-source systems to analyze, discuss, and act on environmental data collectively. EPA data is available to the public through the <a href="https://echo.epa.gov/" target=_blank >Enforcement and Compliance History Online</a> (ECHO) database. ECHO data includes reported violations of environmental laws such as the Clean Air Act (CAA), Clean Water Act (CWA), and the Resource Conservation and Recovery Act (RCRA), which manages hazardous waste, as well as data on emissions, facility inspections, and enforcement actions issued by the EPA and state agencies. However, the database can be challenging to use and error-prone. EEW works to make this data more accessible, user-friendly, and collectively actionable through <a href="/events">public workshops</a>, <a href="https://github.com/edgi-govdata-archiving" target=_blank >open-source data tools</a>, and projects like the <a href="/reports">Congressional District Report Cards</a> and <a href="https://envirodatagov.org/more-permission-to-pollute-the-decline-of-epa-enforcement-and-industry-compliance-during-covid/" target=_blank >COVID-19 enforcement report</a>.
+
+<big>Upcoming Events:</big>
+
+<a href="https://us02web.zoom.us/webinar/register/WN_dAn8xJF9Tqm1aiC1A8Ysig" target=_blank >
+
+  ![Panel Discussion & Report Release October 22 6-7:30pm EST Featuring Union of Concerned Scientists, Hanah Vogel (Policy Advisor to MA Senator Ed Markey), and more](release-event.png)
 
-In these virtual workshops, we will work with community organizers, concerned residents, and university students and staff to dive into the data and stories in your legislative districts, zip codes, and counties to find out: (1) what companies are emitting; (2) what are the associated environmental health risks; (3) what is and isn’t being tracked; (4) whether local facilities are violating their permits; and (5) what environmental enforcement looks like in your area.
+  Register now!
```

Comment:
Consider adding formatting here such as `<big>` and/or `<center>` to make this a clear call to action

Here's how it currently looks:
![Screen Shot 2020-10-10 at 3 30 40 PM](https://user-images.githubusercontent.com/454690/95666380-c7d46d00-0b0d-11eb-9458-0ae0377d9b1a.png)